<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFigDialog</class>
 <widget class="QDialog" name="NewFigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgba(82, 15, 185, 0.25); color: white; font-size: 18px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelNewFigPic">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: blue;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="assets.qrc">:/resource/assets/setAxis.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelDimensionX">
           <property name="text">
            <string>X Dimension</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderDimX">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxDimX">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelDimensionY">
           <property name="text">
            <string>Y Dimension</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderDimY">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxDimY">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelDimensionZ">
           <property name="text">
            <string>Z Dimension</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderDimZ">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxDimZ">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewFigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewFigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxDimZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderDimZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxDimY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderDimY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimX</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBoxDimX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderDimX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
